### ISSUE TITLE
atf presenter with pyplot

### ISSUE THREAD

**Issue by floweisshardt** [floweisshardt/atf#108]
should be able to
- visualize tests results given an `atf_result.txt` file (CI)
  - visualize data and groundtruth for a given metric in a given testblock for a given test, e.g. `time` in `testblock_small` from `atf_test/ts0_c0_r0_e0_s0_0`
  - visualize data for a all metrics in a given testblock for a given test
  - visualize data for all metrics in all testblocks in all tests
  - visualize aggregated data for all test repetitions for a given test, e.g. `atf_test/ts0_c0_r0_e0_s0_0..10`

- visualize comparisons given a single `atf_result.txt` file (Benchmarking)
  - visualize comparison for a given metric in various testblocks of a given test, e.g. `path_length` in testblock `testblock_small` and testblock `testblock_large` from `atf_test/ts0_c0_r0_e0_s0_0`
  - visualize comparision for all repetitions for a given test, e.g. `atf_test/ts0_c0_r0_e0_s0_0..10`

- visualize comparisons given multiple `atf_result.txt` files of the same test package from different test runs(CI-Series)
  - visualize time series data for a given metric in a given testblock for a given test, e.g. `time` in `testblock_small` from `atf_test/ts0_c0_r0_e0_s0_0` from all `atf_result.txt` files.

**Comment by Acuda** [comment]
even if "a given metric in a given testblock for a given test, e.g. time in testblock_small from atf_test/ts0_c0_r0_e0_s0_0" is given on some tests there are multiple "results" due to different parameter sets. 

e.g.:

```
...
     - 
        name: "tf_length_translation"
        started: True
        finished: True
        data: 6.2802
        groundtruth_result: True
        groundtruth_error_message: "all OK"
        groundtruth: 6.283
        groundtruth_epsilon: 0.01
        details: 
          - 
            key: "root_frame"
            value: "world"
          - 
            key: "measured_frame"
            value: "reference1"
      - 
        name: "tf_length_translation"
        started: True
        finished: True
        data: 6.2255
        groundtruth_result: True
        groundtruth_error_message: "all OK"
        groundtruth: 6.283
        groundtruth_epsilon: 0.3
        details: 
          - 
            key: "root_frame"
            value: "world"
          - 
            key: "measured_frame"
            value: "reference1_static" 
...
```

or:

```
...
      - 
        name: "publish_rate"
        started: True
        finished: True
        data: 10.944
        groundtruth_result: True
        groundtruth_error_message: "all OK"
        groundtruth: 11
        groundtruth_epsilon: 5
        details: 
          - 
            key: "topic"
            value: "/topic1"
      - 
        name: "publish_rate"
        started: True
        finished: True
        data: 21.889
        groundtruth_result: True
        groundtruth_error_message: "all OK"
        groundtruth: 22
        groundtruth_epsilon: 5
        details: 
          - 
            key: "topic"
            value: "/topic2"
      
...
      
      - 
        name: "publish_rate"
        started: True
        finished: True
        data: 99.144
        groundtruth_result: True
        groundtruth_error_message: "all OK"
        groundtruth: 99
        groundtruth_epsilon: 5
        details: 
          - 
            key: "topic"
            value: "/topic9"
...
```

how should that handled / visualized?

**Comment by floweisshardt** [comment]
in the end all metrics in a testblock are represented in a list. so we could just loop through the list and plot each element separately. This way it's ok to have multiple publish_rate or path_length metrics in the result.

**Comment by floweisshardt** [comment]
fixed in #140 

Example:
![image](https://user-images.githubusercontent.com/158051/88051891-b2c4ef00-cb59-11ea-9e02-e89f8ea36da8.png)
